src/Protocol/Diaspora.php

```diff
@@ -307,7 +308,7 @@ public static function decodeRaw(string $raw, string $privKey = '', bool $no_exi
 			if ($no_exit) {
 				return false;
 			} else {
-				throw new \Friendica\Network\HTTPException\BadRequestException();
+				throw new BadRequestException();
```

Comment:
We already import `Friendica\Network\HTTPException`
```suggestion
				throw new HTTPException\BadRequestException();
```